cmd/dapr.go

```diff
@@ -30,15 +30,20 @@ var RootCmd = &cobra.Command{
 	Use:   "dapr",
 	Short: "Dapr CLI",
 	Long: `
-	  __                
+	  __
      ____/ /___ _____  _____
     / __  / __ '/ __ \/ ___/
-   / /_/ / /_/ / /_/ / /    
-   \__,_/\__,_/ .___/_/     
-	     /_/            
-									   
+   / /_/ / /_/ / /_/ / /
+   \__,_/\__,_/ .___/_/
+	     /_/
+
 ===============================
 Distributed Application Runtime`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if versionFlag {
+			printVersion()
+		}
+	},
```

Comment:
`--version` flag is only available for root `dapr` command. All other subcommands do not have it. Same a previous versions of CLI.